dorpsgek_github/yaml/keywords/dorpsgek.py

```diff
@@ -0,0 +1,33 @@
+from dorpsgek_github.yaml.exceptions import (
+    YAMLConfigurationError,
+    YAMLDuplicatedDorpsgekCommand,
+)
+from dorpsgek_github.yaml import registry as yaml
+
+
+_registry = {}
+
+
+@yaml.register("dorpsgek")
+def dorpsgek(job, data):
+    """Order DorpsGek to execute certain commands."""
+
+    if data not in _registry.keys():
+        raise YAMLConfigurationError(f"'dorpsgek' has value '{data}'; supported: %s" % ",".join(_registry.keys()))
```

Comment:
oldstyle string formatting

Reply:
Without adding an additional variable, not much to do about it. Using f-string in 90% of the cases doesn't mean the %-style is bad from time to time :) Especially in error-flows :)